modules/admin_manual/pages/installation/docker/index.adoc

```diff
@@ -23,7 +22,42 @@ sudo usermod -aG docker <your-user>
 
 == Quick Evaluation
 
-`sudo docker run -e OWNCLOUD_DOMAIN=localhost:{std-port-http} -p{std-port-http}:{std-port-http} owncloud/server`
+If you only want to take a peek and are content with SQLite as database, which is not supported by ownCloud and not recommended for production purposes, try the following:
```

Comment:
```suggestion
If you only want to take a peek and are content with SQLite as database, which is not supported by ownCloud and only recommended for non-production purposes, try the following:
```
Giving it a more positive expression

Reply:
```suggestion
If you only want to take a peek and are content with SQLite as database, which is not supported by ownCloud for production purposes, try the following:
```

https://doc.owncloud.org/server/10.7/admin_manual/installation/system_requirements.html lists "SQLite (Not for production)" So maybe say this?

Reply:
Perfect.